### ISSUE TITLE
headers format

### ISSUE THREAD

**Issue by its-dibo** [mcollina/autocannon#492]
autocannon docs say that headers can be sent as `-H k=v` but I see that cUrl compatible format can be used `-H k:v`,
I don't know if it is intended or not, but it is better to use the cUrl-compatible format and refer to that in your docs

>  -H/--headers K=V
>        The request headers.

should be changed to
-H/--headers k:V

**Comment by silverwind** [comment]
I agree this `=` also confused me. It should be documented as `"k: v"` format, which is the most common way to pass headers to curl.
